src/Core/Services/Implementations/CollectionService.cs

```diff
@@ -56,11 +56,11 @@ public class CollectionService : ICollectionService
         var usersList = users?.ToList();
 
         // If using Flexible Collections - a collection should always have someone with Can Manage permissions
-        if (_featureService.IsEnabled(FeatureFlagKeys.FlexibleCollections, _currentContext))
+        if (_featureService.IsEnabled(FeatureFlagKeys.FlexibleCollectionsV1, _currentContext))
         {
             var groupHasManageAccess = groupsList?.Any(g => g.Manage) ?? false;
             var userHasManageAccess = usersList?.Any(u => u.Manage) ?? false;
-            if (!groupHasManageAccess && !userHasManageAccess)
+            if (!groupHasManageAccess && !userHasManageAccess && !org.AllowAdminAccessToAllCollectionItems)
```

Comment:
Just clarified with Priya and this error does _not_ depend on `org.AllowAdminAccessToAllCollectionItems`. But at least you've updated the feature flag reference while you're here 😁

Reply:
Understood! Reverted that. Should the other reference to the `FeatureFlagKeys.FlexibleCollections` [here ](https://github.com/bitwarden/server/pull/3530/files#diff-f4e0ad94f5988defbed21fff3fc24821c161ae60bb5ed330ce80ebd7370ce931R46-R47)also be updated to V1?

Reply:
Nope, that reference is used [here](https://github.com/bitwarden/server/blob/5b1a4150388d1d707a0dde0f006328ecdb1b93d4/src/Core/Services/Implementations/CollectionService.cs#L128-L129) which is MVP behaviour, so that's fine.